apps/financial-aid/backend/src/app/modules/application/application.controller.ts

```diff
@@ -16,16 +17,35 @@ import { ApplicationModel } from './models'
 
 import { CreateApplicationDto, UpdateApplicationDto } from './dto'
 
-import { CurrentHttpUser, JwtAuthGuard } from '@island.is/financial-aid/auth'
+import {
+  CurrentHttpUser,
+  JwtAuthGuard,
+  TokenGuard,
+} from '@island.is/financial-aid/auth'
 import type { User } from '@island.is/financial-aid/shared'
 import { ApplicationEventService } from '../applicationEvent'
 
-@UseGuards(JwtAuthGuard)
 @Controller('api')
 @ApiTags('applications')
 export class ApplicationController {
   constructor(private readonly applicationService: ApplicationService) {}
 
+  @UseGuards(TokenGuard)
+  @Get('me')
```

Comment:
We are probably changing this in next PR so feel free to ignore, but does it make sense that the endpoint is `/me` and it returns a value of whether user has applied for month?